src/compositor/lipstickcompositorwindow.cpp

```diff
@@ -297,6 +295,19 @@ void LipstickCompositorWindow::touchEvent(QTouchEvent *event)
     }
 }
 
+void LipstickCompositorWindow::itemChange(QQuickItem::ItemChange change, const QQuickItem::ItemChangeData &data)
+{
+    if (change == ItemSceneChange) {
+        if (window())
+            window()->removeEventFilter(this);
+
+        if (data.window)
+            data.window->installEventFilter(this);
```

Comment:
Not that we use it, but on line 286, there is an envvar check to not install the event filter. Might want to move that one here.